WcaOnRails/package.json

```diff
@@ -52,6 +52,6 @@
     "webpack-dev-server": "^3.2.1"
   },
   "scripts": {
-    "webpack:analyze": "mkdir -p public/packs && NODE_ENV=production node_modules/.bin/webpack --config config/webpack/production.js --profile --json > public/packs/stats.json && node_modules/.bin/webpack-bundle-analyzer public/packs/stats.json"
+    "webpack:analyze": "mkdir -p public/packs && RACK_ENV=test NODE_ENV=production webpack --config config/webpack/production.js --profile --json > public/packs/stats.json && webpack-bundle-analyzer public/packs/stats.json"
```

Comment:
Hum wouldn't that only work if `node_modules/bin` is in your `PATH`?
(or if webpack was installed "globally" on the computer)

Reply:
Ah, npm/yarn is actually nice enough to set up your `PATH` for you when you run a script! See https://docs.npmjs.com/misc/scripts#path. This works for me and I do *not* have `node_modules/bin` in my `PATH`, nor do I have webpack installed globally =)

Reply:
Awesome!